ambry-account/src/main/java/com/github/ambry/account/MySqlAccountService.java

```diff
@@ -199,20 +199,17 @@ public Account getAccountByName(String accountName) {
   }
 
   @Override
-  public boolean updateAccounts(Collection<Account> accounts) throws AccountServiceException {
+  public void updateAccounts(Collection<Account> accounts) throws AccountServiceException {
     Objects.requireNonNull(accounts, "accounts cannot be null");
     if (accounts.isEmpty()) {
-      logger.debug("Empty account collection to update.");
-      return false;
+      throw new IllegalArgumentException("Empty account collection to update.");
```

Comment:
nit:  This can be `BadRequest` AccountServiceErrorCode. (We can do this in future PR)

Reply:
ok